rpc/network/network.go

```diff
@@ -193,9 +203,18 @@ func (nm *Manager) UpdateRelatedChainID(chainID uint64, relatedChainID uint64) e
 	return err
 }
 
-func (nm *Manager) UpdateRPCURLs(chainID uint64, rpcURL, originalRPCURL, fallbackURL, originalFallbackURL string) error {
-	_, err := nm.db.Exec(`UPDATE networks SET rpc_url = ?, original_rpc_url = ?, fallback_url = ?, original_fallback_url = ?  WHERE chain_id = ?`,
-		rpcURL, originalRPCURL, fallbackURL, originalFallbackURL, chainID)
+func (nm *Manager) UpdateRPCURL(chainID uint64, rpcURL string) error {
```

Comment:
@alaibe guess that we don't need to make this and the calls below public (they all should start with a small first letter)?

Reply:
Done